openeo/internal/jupyter.py

```diff
@@ -105,7 +105,7 @@ def render_component(component: str, data = None, parameters: dict = {}):
     """.format(
         script = SCRIPT_URL,
         component = component,
-        props = json.dumps(parameters)
+        props = json.dumps(parameters, default = lambda o: o.__dict__) # default parameter added to serialize LogEntry
```

Comment:
This `default = lambda o: o.__dict__` basically means "export everything" and is pretty liberal. I'm a bit worried whether this will cause accidental exposure of things you don't want to expose (not now, but in the longer term when Jupyter support will expand).

As far as I understand this is just to make LogEntry items JSON-seriazable, right?

Reply:
Yes.

Reply:
I pushed an alternative implementation that only does the `o.__dict__` on `LogEntry`  instances